src/Components/Facility/FacilityCreate.tsx

```diff
@@ -467,6 +468,7 @@ export const FacilityCreate = (props: FacilityProps) => {
               <div>
                 <InputLabel id="name-label">Facility Name*</InputLabel>
                 <TextInputField
+                  data-test="facility-name"
```

Comment:
```suggestion
                  id="facility-name"
```

Why can't we add this as Id for the input? 
The label should have an `htmlFor`